src/lib/model/ContentTypeSchema.ts

```diff
@@ -0,0 +1,44 @@
+import { HalResource } from '../hal/models/HalResource';
+import { Page } from './Page';
+
+export interface ContentTypeCachedSchema {
```

Comment:
Missing comment block

Reply:
added in some supporting comments

Reply:
Missing comment block for the `ContentTypeCachedSchema` interface too